micrometer-tracing-bridges/micrometer-tracing-bridge-otel/src/test/java/io/micrometer/tracing/otel/bridge/OtelTracingApiTests.java

```diff
@@ -69,10 +73,18 @@ class OtelTracingApiTests {
     // You can consider
     // customizing the baggage manager with correlation and remote fields (currently
     // we're setting empty lists)
-    OtelTracer tracer = new OtelTracer(otelTracer, otelCurrentTraceContext, event -> {
-        slf4JEventListener.onEvent(event);
-        slf4JBaggageEventListener.onEvent(event);
-    }, new OtelBaggageManager(otelCurrentTraceContext, Collections.emptyList(), Collections.emptyList()));
+    OtelTracer tracer;
+
+    {
```

Comment:
I'm not sure a static initializer is a good idea here, especially because this code is part of the documentation: https://micrometer.io/docs/tracing#_micrometer_tracing_opentelemetry_setup

Reply:
@marcingrzejszczak Could you please help here?
I'm thinking we might need unit tests for`Slf4JEventListener`.

Reply:
> I'm not sure a static initializer is a good idea here, especially because this code is part of the documentation: https://micrometer.io/docs/tracing#_micrometer_tracing_opentelemetry_setup

but if we don't do this,`Slf4JEventListener`  won't work

Reply:
I was not saying we should not register it, I'm saying we should do it diferently.

Reply:
> I was not saying we should not register it, I'm saying we should do it differently.

Agree!